src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -805,14 +805,20 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         }
 
         /// <summary>
-        /// Create a <see cref="LinearSvmTrainer"/>, which predicts a target using a linear binary classification model trained
+        /// Create <see cref="LinearSvmTrainer"/>, which predicts a target using a linear binary classification model trained
         /// over boolean label data.
         /// </summary>
         /// <param name="catalog">The binary classification catalog trainer object.</param>
         /// <param name="labelColumnName">The name of the label column. The column data must be <see cref="System.Boolean"/>.</param>
         /// <param name="featureColumnName">The name of the feature column. The column data must be a known-sized vector of <see cref="System.Single"/>.</param>
         /// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>
-        /// <param name="numberOfIterations">The number of training iteraitons.</param>
+        /// <param name="numberOfIterations">The number of training iteration.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[LinearSvm](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LinearSvm.cs)]
```

Comment:
>LinearSvm [](start = 128, length = 9)

please double check the path, so that we don't get new bad ref issues.

Reply:
Did check.

---
In reply to: [278372284](https://github.com/dotnet/machinelearning/pull/3575#discussion_r278372284) [](ancestors = 278372284)